spec/lib/krikri/enrichments/split_coordinates_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+describe Krikri::Enrichments::SplitCoordinates do
+  it_behaves_like 'a field enrichment'
+
+  context 'with place object' do
+    let(:lat) { nil }
+    let(:long) { nil }
+    let(:place) do
+      build(:place, lat: lat, long: long)
+    end
+
+    shared_examples 'latitude and longitude' do |expected_lat, expected_long|
+      it 'assigns latitude' do
+        expect(subject.enrich_value(place).lat).to eq expected_lat
+      end
+      it 'assigns longitude' do
+        expect(subject.enrich_value(place).long).to eq expected_long
+      end      
+    end
+
+    shared_examples 'nonsensical coordinates' do
+      it 'returns empty coordinates' do
+        rv = subject.enrich_value(place)
+        expect(rv.lat).to eq []
```

Comment:
I'm now thinking that this should retain the original values. I don't know if we want to delete coordinates that we can't recognize, but if/when we do, that seems like another enrichment. 

Thoughts @anarchivist?